docs/source/quick-start.rst

```diff
@@ -54,20 +54,24 @@ The standard `OutOfRangeNullTransformer <https://tubular.readthedocs.io/en/lates
 Dates
 ^^^^^
```

Comment:
All these additions are fine but there are a few other errors/improvments in the quickstart guide that it might be worth fixing in this tidy up:

- In Numeric and Nominal transformers inconsistently have brackets around the links
- in "Installation" I think "important" is meant to format the text but is appearing as text
- I think it would look a lot nicer with tubular in the first line as text with the picture underneath, rather than trying to use the logo as text